mbf_abstract_nav/src/abstract_navigation_server.cpp

```diff
@@ -334,15 +335,18 @@ mbf_abstract_nav::AbstractControllerExecution::Ptr AbstractNavigationServer::new
     const std::string name,
     const mbf_abstract_core::AbstractController::Ptr plugin_ptr)
 {
-  return boost::make_shared<mbf_abstract_nav::AbstractControllerExecution>(name, plugin_ptr, vel_pub_, goal_pub_,
-      tf_listener_ptr_, last_config_, boost::function<void()>(), boost::function<void()>());
+//  return boost::make_shared<mbf_abstract_nav::AbstractControllerExecution>(nh_, private_nh_, name, plugin_ptr, vel_pub_, goal_pub_,
```

Comment:
Why replacing `boost::make_shared` here ? (same in `costmap_controller_execution.cpp`)

Reply:
`boost::make_shared` is limited to 9 constructor arguments, but with the two additional arguments this limit is exceeded.

C++11 introduced variadic templates to overcome this limitation, and Boost should apply this if available. But for some reason I got compilation errors and did not look into details so far. Maybe Boost 1.58 in Ubuntu Xenial does not have the conditional C++11 extensions yet.